### PR TITLE
Adds newrelic, artillery, artillery yml file, and http proxy middlewa…

### PR DESCRIPTION
**Pull request by sharmoo**
…re to set up proxy for service

Fire.yml consists of the artillery testing. Newrelic.js is the default configuration file with two changes (app name and license). 
index.js has been changed to use http proxy middleware in order to serve the service.

### CODE REVIEW

File: fire.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            comment: "Second."
+            commenttime: 0
+            username: "sharmoo"
+      - delete:
+            url: "/song/{{ songId }}/api/song_comment"
+            json:
+              id: "{{ commentId }}"
+
+      - delete:
+            url: "/song/{{ songId }}/api/song_id"
```
Comment (JonBeyond) [inline fire.yml]: Artillery is only supposed to be stress testing your use case (which are GETs) .  It's probably why you mentioned you are getting low numbers.

File: index.js [hunk 2]
```diff
@@ -1,15 +1,21 @@
+require('newrelic');
+
 const express = require('express');
 const morgan = require('morgan');
```
Comment (JonBeyond) [inline index.js]: remove morgan, it is middleware that slows down your server. should only be used for dev

Comment (JonBeyond) [reply index.js]: (remember that console.logs are very slow operations)

File: newrelic.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['ProxyPlayer'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '9e5220ad574f2754a88dec753edbf1d7a27bdfa7',
```
Comment (JonBeyond) [inline newrelic.js:16]: you need to add `newrelic.js` to your `.gitignore`

**Review by JonBeyond — CHANGES_REQUESTED** [review]
Please correct your artillery test file to represent your use case
Add newrelic.js to your gitignore
Remove morgan

Tag me once you've pushed a new commit and I will review/approve

File: index.js (hunk 2 above)
Comment (sharmoo) [reply index.js]: Excellent point! Thank you. Resolved

File: newrelic.js (hunk 3 above)
Comment (sharmoo) [reply newrelic.js:16]: Oh shoot. Nice catch. Done!

File: fire.yml (hunk 1 above)
Comment (sharmoo) [reply fire.yml]: Done!

**Review by JonBeyond — APPROVED** [review]

**Pull request merged**